src/core/build_target.go

```diff
@@ -115,6 +115,8 @@ type BuildTarget struct {
 	// Maps the original declaration to whatever dependencies actually got attached,
 	// which may be more than one in some cases. Also contains info about exporting etc.
 	dependencies []depInfo `name:"deps"`
+	// The (transitive) run-time dependencies of this target.
```

Comment:
This comment reads to me that this field will be expanded to include the complete set of transitive runtime dependencies, which I don't think is actually the case. I think the term "transitive" is not one to be used lightly, and the comments where you are using it need to be very explicit about what that means (or not use it at all).

Reply:
Fair enough - the transitivity property is handled by `IterAllRuntimeDependencies`, so I can see how this is confusing.